### PR TITLE
Add fsfreeze status to vmi object

### PR DESCRIPTION
**Pull request by ShellyKa13**
**What this PR does / why we need it**:
This PR updates the VMI object with the guest fsfreeze status in case the fs is frozen
The vmi launcher polls from the guest using the guest agent for the fsfreeze and updates the VMI if it is frozen

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/kubevirt.io/client-go/api/v1/types.go [hunk 1]
```diff
@@ -211,6 +211,11 @@ type VirtualMachineInstanceStatus struct {
 	// +optional
 	// +listType=atomic
 	VolumeStatus []VolumeStatus `json:"volumeStatus,omitempty"`
+
+	// FSFreezeStatus is the state of the fs of the guest
+	// it can be either frozen or thawed
+	// +optional
+	FSFreezeStatus string `json:"fsFreezeStatus,omitempty"`
```
Comment (mhenriks) [inline staging/src/kubevirt.io/client-go/api/v1/types.go:218]: If this is optional, should probably be `*string`

Comment (xpivarc) [reply staging/src/kubevirt.io/client-go/api/v1/types.go:218]: Why is it needed here when it will be in `GuestOSInfo`?

Comment (ShellyKa13) [reply staging/src/kubevirt.io/client-go/api/v1/types.go:218]: @mhenriks from what I see all the other optional variables in this struct which are string are not pointers..
@xpivarc  it is not in the GuestOSInfo, its in the guest agent info which is not shown in the vmi information

File: pkg/virt-launcher/virtwrap/agent-poller/agent_parser_test.go [hunk 2]
```diff
@@ -257,6 +257,16 @@ var _ = Describe("Qemu agent poller", func() {
 			Expect(err).To(HaveOccurred(), "Should not parse the info")
 		})
 
+		It("should parse FSFreezeStatus", func() {
```
[the remark this answers is not in the document]
Comment (ShellyKa13) [reply pkg/virt-launcher/virtwrap/agent-poller/agent_parser_test.go:260]: done

File: staging/src/kubevirt.io/client-go/api/v1/types.go (hunk 1 above)
Comment (mhenriks) [reply staging/src/kubevirt.io/client-go/api/v1/types.go:218]: @ShellyKa13 okay consistency is good

**Pull request merged**